plugin/evm/atomic_trie_test.go

```diff
@@ -591,6 +597,187 @@ func TestApplyToSharedMemory(t *testing.T) {
 	}
 }
 
+func Test_atomicTrie_AcceptTrie(t *testing.T) {
```

Comment:
nit: 
```suggestion
func TestAtomicTrie_AcceptTrie(t *testing.T) {
```
seems more consistent with other test names

Reply:
Let's have it as

```suggestion
func TestatomicTrie_AcceptTrie(t *testing.T) {
```

? Just in case we later have an exported `AtomicTrie` to differentiate the tests

Reply:
I don't think that reads well :(

Reply:
Changed to TestAtomicXXX